src/components/ShareLinks/ShareLinksPresentation.js

```diff
@@ -5,7 +5,16 @@ import copyToClipboard from 'common/utils/copyToClipboard'
 // FPCC
 import getIcon from 'common/utils/getIcon'
 import { useNotification } from 'context/NotificationContext'
-function ShareLinksPresentation({ url, title, modalCloseHandler }) {
+import { QrcodeButton } from 'components/Actions'
+import { makePlural } from 'common/utils/urlHelpers'
+
+function ShareLinksPresentation({
+  url,
+  sitename,
+  title,
+  modalCloseHandler,
+  entry,
```

Comment:
I see that you've added some new props to ShareLinksPresentation (entry and sitename). You will need to pass these in wherever ShareLinksPresentation is being used.

Reply:
I *think* its all fixed now :)